docs/topics/api/addons.rst

```diff
@@ -284,6 +284,67 @@ This endpoint allows you to fetch a single version belonging to a specific add-o
     :>json string version: The version number string for the version.
 
 
+-----------------
+Review Notes List
+-----------------
+
+.. _review-notes-all:
+
+This endpoint allows you to list all the approval/rejection review history for a specific add-on, per version.
+
+.. http:get:: /api/v3/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/reviewnotes/
+
+    .. note::
+        All add-ons require authentication and either
+        reviewer permissions or a user account listed as a developer of the
+        add-on.
+
+    :>json int count: The number of versions for this add-on.
+    :>json string next: The URL of the next page of results.
+    :>json string previous: The URL of the previous page of results.
+    :>json array results: An array of :ref:`per version review notes<review-version-object>`.
+
+
+---------------------------
+Review Notes Version Detail
+---------------------------
+
+.. _review-notes-version:
+
+This endpoint allows you to fetch the review notes for a single version belonging to a specific add-on.
+
+.. http:get:: /api/v3/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/reviewnotes/(int:id)/
+
+    .. _review-version-object:
+
+    :>json int id: The version id.
+    :>json string version: The version number string for the version.
+    :>json array notes: Array holding the individual notes for the version.
+    :>json int notes[].id: The id for a review note.
+    :>json string notes[].action: The :ref:`type of review note<review-note-action>`.
+    :>json string notes[].action_label: The text label of the action.
+    :>json int notes[].user: The numeric user id of the reviewer or author who left the review note.
+    :>json string notes[].comments: The text content of the review note.
+    :>json string notes[].date: The date the review note was created.
+
+
+.. _review-note-action:
+
+    Possible values for the ``action`` field:
+
+    ==========================  ==========================================================
+                         Value  Description
+    ==========================  ==========================================================
+                      approved  Version, or file in the version, was approved
+                      rejected  Version, or file in the version, was rejected
+              review-requested  Developer requested review
+    more-information-requested  Reviewer requested more information from developer
+        super-review-requested  Add-on was referred to an admin for attention
+                       comment  Reviewer added comment for other reviewers
+                   review-note  Generic review comment 
+    ==========================  ==========================================================
+
```

Comment:
I didn't add any, so unless there a default in `ModelSerializer` or one of the viewset classes, no.